desktop_version/src/Screen.cpp

```diff
@@ -363,6 +363,12 @@ void Screen::recacheTextures(void)
     }
 }
 
+#ifdef __ANDROID__
+bool Screen::isForcedFullscreen(void)
+{
+    return true;
```

Comment:
We can probably move this def to _inside_ the function, so that we don't accidentally end up with symbol conflicts if this function prototype ever changes.